lighthouse-core/fraggle-rock/gather/session.js

```diff
@@ -13,10 +13,10 @@ const DEFAULT_PROTOCOL_TIMEOUT = 30000;
 /** @implements {LH.Gatherer.FRProtocolSession} */
 class ProtocolSession {
   /**
-   * @param {LH.Puppeteer.CDPSession} session
+   * @param {LH.Puppeteer.CDPSession} cdpSession
```

Comment:
Maybe `pptrSession`?

Reply:
I'd be on board with `pptrSession` if this was external facing, but constructing these should (hopefully) always be internal. I'm +1 to `cdpSession` to match the class name in that case.

Reply:
Yeah I'm happy with both. Let's change it in the future if we keep referring to it as the pptr session.